yarn-project/circuit-types/src/logs/l1_payload/l1_event_payload.ts

```diff
@@ -7,6 +8,12 @@ import { EncryptedEventLogIncomingBody } from './encrypted_log_incoming_body/ind
 import { L1Payload } from './l1_payload.js';
 import { Event } from './payload.js';
 
+function isEventTypeIdValid(eventTypeId: Fr): boolean {
+  const buf = eventTypeId.toBuffer();
+  // Check that the first 28 bytes are zero and the value itself is non-zero
```

Comment:
Might just being annoying here, but why is zero not acceptable value? I get why we are checking the first 28 bytes, because they need to be zero.

Reply:
It is now accepted.